src/utils/string.js

```diff
@@ -19,16 +19,32 @@ const REMOVE_URLS_REGEXP = (function () {
     'moz-page-thumb',
   ];
 
+  // Captures the protocol part (like "http://") in group 1
+  const standardUrlPattern = `\\b((?:${protocols.join('|')})://)/?[^\\s/$.?#][^\\s)]*`;
+  //                          ^  ^                              ^ ^          ^
+  //                          |  |                              | |          Matches any characters except
+  //                          |  |                              | |          whitespaces and ')' character.
+  //                          |  |                              | Matches any character except whitespace
+  //                          |  |                              | and '/', '$', '.', '?' or '#' characters
+  //                          |  |                              | because this is start of the URL path/host
+  //                          |  |                              Optional '/' after '://'
+  //                          |  Captures the protocol and '://' part (Group 1)
+  //                          Word boundary, ensures the protocol isn't part of a larger word.
+
+  // Captures the base 'about:...' part (like "about:profiling") in group 2
+  const aboutQueryPattern = `\\b(about:[^?#\\s]+)([?#])[^\\s]*`;
```

Comment:
optional:
note that `\S` is equivalent to `[^\s]` so you could do:
```suggestion
  const aboutQueryPattern = `\\b(about:[^?#\\s]+)([?#])\\S*`;
```

Reply:
Oh, while looking at this I noticed a bug in the implementation actually. This should be similar to what we have in the `standardUrlPattern`. In there we have:
```
[^\\s)]
```
Note the `)` character there. This is to make sure that we don't capture the end parenthesis in strings like `text (https:/example.com)` So the result should be `text (<URL>)` We have these strings in javascript frames especially. This `aboutQueryPattern` pattern doesn't handle that. I've changed the code to `[^\\s)]` handle it as well.